react/features/toolbox/components/VideoMuteButton.js

```diff
@@ -191,13 +187,8 @@ class VideoMuteButton extends AbstractVideoMuteButton<Props, *> {
 function _mapStateToProps(state): Object {
     const { enabled: audioOnly } = state['features/base/audio-only'];
     const tracks = state['features/base/tracks'];
-    let _videoMuted = isLocalVideoTrackMuted(tracks);
-    let _videoDisabled = false;
-
-    if (isPrejoinPageVisible(state)) {
-        _videoMuted = isPrejoinVideoMuted(state);
-        _videoDisabled = isPrejoinVideoDisabled(state);
-    }
+    const _videoMuted = isLocalVideoTrackMuted(tracks);
+    const _videoDisabled = isPrejoinPageVisible(state) ? isPrejoinVideoDisabled(state) : false;
```

Comment:
As commented above, I think this should follow the same logic as the audio button.

Reply:
Fixed.